### ISSUE TITLE
Support better control over logging

### ISSUE THREAD

**Issue by rahulsom** [mpurbo/MMPReactiveStompClient#2]
Currently, MMPRxSC_LOG is defined to use NSLog if DEBUG is defined. 

That produces too much noise. I use CocoaLumberjack in my app, but that would be pushing for too much in this library.

It would be nice if logging could be controlled from a macro within my app.

**Comment by mpurbo** [comment]
Yes I agree, it's very noisy. Using a full logging framework is probably not a very good solution either.  Since it's based on ReactiveCocoa, one of the idea I have is to have log messages available as signals. Something like:

``` objc
+ (RACSignal *)debugMessages;
+ (RACSignal *)warningMessages;
+ (RACSignal *)errorMessages;
```

This way, you could subscribe to debug messages and log it only if you want them, e.g.

``` objc
[[MMPReactiveStompClient debugMessages] subscribeNext:^(NSString *message) { 
    NSLog(@"[DEBUG] %@", message);
}]
```
